packages/core/parcel-bundler/src/SourceMap.js

```diff
@@ -38,7 +38,13 @@ class SourceMap {
   }
 
   async addMap(map, lineOffset = 0, columnOffset = 0) {
+    if (map.toJSON) {
```

Comment:
When is this the case?
Parcel never creates a sourcemapGenerator (as the SourceMap class is both a Generator and Consumer and faster) or does any css asset return a sourceMapGenerator instead of a sourceMap?

Reply:
> Parcel never creates a sourcemapGenerator

PostCSS does...

Reply:
Ow alright didn't expect that.
Perhaps better to move that transforming to a standard sourcemap to where you get the result from PostCSS? (See later comment)

Reply:
This need to happen in `CSSAst.render` and transforms/postcss. This way, I tried to prevent duplicating the filter part.